src/shared/components/lazyMobXTable/LazyMobXTable.tsx

```diff
@@ -910,10 +910,7 @@ export default class LazyMobXTable<T> extends React.Component<
     }
 
     protected updateColumnVisibility(id: string, visible: boolean) {
-        // ignore undefined columns
-        if (this.store.columnVisibility[id] !== undefined) {
```

Comment:
@onursumer Is there a reason we need to "ignore undefined columns"?

Reply:
Hmm, I can't remember the exact reason but seems okay to remove this.